src/vs/workbench/contrib/welcomeGettingStarted/common/gettingStartedContent.ts

```diff
@@ -385,6 +385,14 @@ export const walkthroughs: GettingStartedWalkthroughContent = [
 						type: 'svg', altText: 'VS Code Settings', path: 'settings.svg'
 					},
 				},
+				{
+					id: 'profiles',
+					title: localize('gettingStarted.profiles.title', "Customize your workspace with Profiles"),
```

Comment:
I would not mention the word "workspace" here, since profiles help with VS Code customisation.
For example, "Customise VS Code with Profiles".

Not sure if "VS Code" we need to get from the product.json, or it is fine to hard code that name.

Reply:
Suggested to get the product name from product.json

Reply:
We reference VS Code in a few of the walkthrough content here. I create another issue to fix it everywhere: https://github.com/microsoft/vscode/issues/185602